src/duties/autoFix.js

```diff
@@ -1,42 +1,42 @@
-let _ = require('lodash/fp')
-let Git = require('nodegit')
-let path = require('path')
-let { execSync } = require('child_process')
-let { Repository } = Git
-let { getRunningDirectory } = require('../utils')
+let _ = require("lodash/fp");
```

Comment:
why are there double quotes and semicolons here? looks like the prettier config messed up